src/Mod/Sketcher/Gui/Resources/Sketcher.qrc

```diff
@@ -214,6 +214,7 @@
     </qresource>
     <qresource>
         <file>icons/pointers/Sketcher_Pointer_CarbonCopy.svg</file>
+        <file>icons/pointers/Sketcher_Pointer_Copy.svg</file>
```

Comment:
Unless @abdullahtahiriyo  is against it, the Copy/clone tools are going to be replaced by the translate tool (https://github.com/FreeCAD/FreeCAD/pull/11267). So I think you can remove the icon if Abdullah confirm he's ok with my plan.

Reply:
I'll leave them in but can be removed if the new tool is merged

Reply:
FreeCAD may need to be used after this is merged and before the other is merged. It is always best to make merge decisions independent (as much as it can be done). Good call to keep it.

Reply:
Note that this PR introduce the icon, but if I understood correctly the icon is unused because in this case the code is using a hardcoded XPM. Which is why I suggested dropping it. But it can indeed be left, but then it would make sense to ditch the XPM and use the icon.

Reply:
I'll remove them

Reply:
Thanks for the clarification. I am ok with removing.